### DISCUSSION TITLE
Identifiers are not quoted when using i.e. DropConstraint(), AddConstraint(), or CreateColumn()

### DISCUSSION THREAD

**Discussion by nxp10225** [sqlalchemy/sqlalchemy#10660] [Usage Questions]
I figured it out. Turns out that 'if' isn't a reserved word, and that the " vs ` quoting has to do with the sql_mode. Which I still need to figure out.

**Comment by CaselIT** [comment]
Hi, 

in general it depends on the dialect in use if a word is reserved or now. using `str(thing)` or `thing.compile()` will use the default dialect
